upgrade-notes/backend_20250206_212223.md

```diff
@@ -0,0 +1,13 @@
+#### add product recalculation queue deduplication ([#3669](https://github.com/shopsys/shopsys/pull/3669))
+
+- ensure you have Redis on version 7.4 (see #3673)
+- method use `\Shopsys\FrameworkBundle\Model\Product\ProductFacade::iterateAllProductIds()` was removed, use `iterateAllProductIdsExceptVariant()`instead
+- method use `\Shopsys\FrameworkBundle\Model\Product\ProductRepository::iterateAllProductIds()` was removed, use `iterateAllProductIdsExceptVariant()`instead
+- `ProductRecalculationMessage` messages no longer contain `exportScopes`. Scopes are now stored in Redis.
```

Comment:
maybe it is safer to mention the particular classes(s) - `AbstractProductRecalculationMessage` and its descendants - `ProductRecalculationPriorityHighMessage` and `ProductRecalculationPriorityRegularMessage`

Reply:
6a8f07b139